dist/pom.xml

```diff
@@ -259,7 +259,7 @@
 
     <dependency>
       <groupId>io.swagger.core.v3</groupId>
-      <artifactId>swagger-annotations</artifactId>
+      <artifactId>swagger-annotations-jakarta</artifactId>
```

Comment:
❓ Why we had to do this?

Reply:
The maven dependency plugin "complained" about `swagger-annotations` and `swagger-annotations-jakarta` by switching around dependent on which one was added to the `pom.xml`. Since the stack uses Jakarta anyway, I pinned it to `-jakarta`. Also, that way we avoid having both dependencies in the distribution on the classpath.